spec.html

```diff
@@ -20239,6 +20794,18 @@ <h1>Runtime Semantics: EvaluateBody</h1>
           1. Perform ! Call(_promiseCapability_.[[Reject]], *undefined*, &laquo; _declResult_.[[Value]] &raquo;).
         1. Return Completion { [[Type]]: ~return~, [[Value]]: _promiseCapability_.[[Promise]], [[Target]]: ~empty~ }.
       </emu-alg>
+
+      <emu-grammar>
+        Initializer :
+          `=` AssignmentExpression
+      </emu-grammar>
+      <emu-alg>
+        1. Assert: _argumentsList_ is empty.
+        1. Let _exprRef_ be the result of evaluating |AssignmentExpression|.
+        1. Let _exprValue_ be ? GetValue(_exprRef_).
+        1. Return Completion { [[Type]]: ~return~, [[Value]]: _exprValue_, [[Target]]: ~empty~ }.
```

Comment:
would this work?
```suggestion
        1. Return NormalCompletion(_exprValue_).
```

Reply:
I don't think it would. This is called from DefineField → Call → [the [[Call]] internal method for ECMAScript function objects](https://tc39.es/ecma262/#sec-ecmascript-function-objects-call-thisargument-argumentslist), which does

1. Let result be OrdinaryCallEvaluateBody(F, argumentsList).
1. If result.[[Type]] is return, return NormalCompletion(result.[[Value]]).
1. ReturnIfAbrupt(result).
1. Return NormalCompletion(undefined). 

so the value in a NormalCompletion would be dropped.